docs/migrating-from-other-versions.md

```diff
@@ -0,0 +1,69 @@
+# Migrating From Other Versions
+----
+This client is an improved version of [a previous one](https://github.com/logdna/nodejs).
```

Comment:
would this file be a permanent addition to the readme- or just temporary- we could also make note of this in the readme/changelog

Reply:
Which part?  The part about old versions?  I've seen other packages talk about "migrating" at the top, and I've found that very up front and helpful.  I think the more we can disclose about our "new and old" versions, the less confusing it will be for people.

Can you explain what you mean?

Reply:
I agree that we should be transparent about migrating repositories- I just wonder how long this particular file will stick around- and if there is another way to communicate this either in our README and CHANGELOG

Reply:
Oh, well it's already linked from the main README. I don't think the change log is as visible. We can always remove it down the road if migrating is no longer relevant.